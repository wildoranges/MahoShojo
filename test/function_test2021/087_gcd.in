13
32860 47240
34234 9291
3243 821
984201 892342
842 102
32 918
4 8
255 1087
44 229
5423 260
35 7
4269 4213
556 208
