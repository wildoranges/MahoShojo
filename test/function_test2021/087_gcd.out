20
1
1
1
2
2
4
1
1
1
7
1
4
0
